### PR TITLE
Changed label

### PR DESCRIPTION
**Pull request by chosta**
#1141
Changed the label to WGT as discussed in discord.
We need someone to update the wiki too. I'll add the proper link to the (i) button once this is done.

Edit. We can leave the wiki as it is. Merge whenever

### CODE REVIEW

**Comment by nwai90** [comment]
@chosta Could you replace the three instances of "Use solar phase" in the respective `ui.json` files as well?

**Comment by nwai90** [comment]
Updated header to https://github.com/bafolts/terraforming-mars/wiki/Variants#world-government-terraforming

**Comment by nwai90** [comment]
@alrusdi Let's merge this if the translation is ok?

File: src/locales/ru/ui.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "Prelude": "Прологи",
     "Venus Next": "Венера",
     "Custom Corporation list": "Настроить корпорации",
-    "Use Solar Phase": "Помощь Солнечной системы",
+    "World Government Terraforming": "Терраформирование мирового правительства",
```
Comment (alrusdi) [inline src/locales/ru/ui.json:31]: It sounds like "Правительственная поддержка" in original Russian rules

Comment (chosta) [reply src/locales/ru/ui.json:31]: Sorry, my russian is a bit rusty these days :)

**Pull request merged**